### PR TITLE
flake8 fixes

### PR DESCRIPTION
**Pull request by Aniket21mathur**
Fixes https://github.com/sugarlabs/chat/issues/25

### CODE REVIEW

File: activity.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
     gi.require_version('Gst', '1.0')
     from gi.repository import Gst
     _HAS_SOUND = True
-except:
+except BaseException:
```
Comment (pro-panda) [inline activity.py]: This does not indicate why is the `try` clause being used.
Use `ImportError` instead of `BaseException`

**Comment by Aniket21mathur** [comment]
>     * Replace all instances of `BaseException` with a more appropriate exception.

Done.



>     * Do not include reference to github issues in commit messages

Done. Thanks!

File: utils.py [hunk 2]
```diff
@@ -61,5 +61,5 @@ def _get_initial_value(self):
                                       'EV_SW', 'SW_TABLET_MODE'])
             # 10 is ebook_mode, 0 is normal
             return (output == 10)
-        except:
+        except BaseException:
```
Comment (pro-panda) [inline utils.py]: One `BaseException` instance still remaining

Comment (Aniket21mathur) [reply utils.py]: Thanks. Updated.

**Comment by quozl** [comment]
Thanks.  Merged by hand https://github.com/sugarlabs/chat/commit/1206776a2375b8911144bded98992afaac1ce73b.

**Pull request closed without merging**